### PR TITLE
Update the canvas/graphics state and UI code

### PR DESCRIPTION
**Pull request by bvaughn**
- [x] Replace older, non-standard cache for fetching paint graphics with standard Suspense-compatible cache
- [x] Removed unused `<video>` tag and all references to it in Redux
- [ ] Replace imperative screenshot loading code in `protocol/graphics` and Redux with a new React component
  - [x] Fetch and display imprecise and precise screenshots
  - [x] Add better loading, stalled, and error UX
  - [ ] Support comment overlay and click-to-add-comment interactions
  - [ ] Support Node picker and click-to-select interactions
  - [ ] Support DOM highlight
  - [ ] Figure out how to support playback (Suspending to fetch doesn't work, because of "starvation")

### CODE REVIEW

File: src/ui/components/VideoNew.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        <img
+          className={styles.Image}
+          src={`data:${screenShotToRender.mimeType};base64,${screenShotToRender.data}`}
+          onClick={onClick}
+          onContextMenu={onContextMenu}
+        />
+      )}
+
+      {/* TODO [FE-2104] Add comment overlay */}
+      {/* TODO [FE-2104] Add DOM element highlight overlay */}
```
Comment (jasonLaster) [inline src/ui/components/VideoNew.tsx:80]: 😉

File: packages/protocol/graphics.ts [hunk 2]
```diff
@@ -191,41 +183,9 @@ export function setupGraphics(store: AppStore) {
...
-        paintGraphics(screen, mouse);
-      }
-
-      if (typeof onPausedAtTime === "function") {
-        onPausedAtTime(time);
-      }
-
-      await repaint();
-    },
-  });
```
Comment (jasonLaster) [inline packages/protocol/graphics.ts:225]: I love seeing this complexity go away

File: src/ui/actions/timeline.ts [hunk 3]
```diff
@@ -185,29 +184,10 @@ export async function getInitialPausePoint(recordingId: string) {
...
-      const stateAfterScreenshot = getState();
-
-      if (getHoverTime(stateAfterScreenshot) !== time) {
-        return;
-      }
-
-      paintGraphics(screen, mouse);
-    } catch (error) {
-      console.error(error);
-    }
```
Comment (jasonLaster) [inline src/ui/actions/timeline.ts:210]: Love killing this

**Review by jasonLaster — COMMENTED** [review]
This is removing some of our sketchiest code

**Comment by bvaughn** [comment]
Migrating this work over to #10255 which has been rebased on main and includes all of the incremental changes split off of this larger, older PR.

**Pull request closed without merging**